package.json

```diff
@@ -1,8 +1,15 @@
 {
   "name": "@asteasolutions/zod-to-openapi",
-  "version": "6.4.0",
+  "version": "7.0.0",
```

Comment:
There is no need to modify those manually. We have an automated release process and we'll update them accordingly

Reply:
Fixed